desk/app/groups.hoon

```diff
@@ -584,12 +671,30 @@
       %-  (slog leaf/"Error forwarding poke" u.p.sign)
       go-core
     ==
+  ++  go-take-epic
+    |=  her=epic:e
+    ^+  go-core
+    ?>  ?=(%sub -.net)
+    ?:  =(her okay)
+      go-core
+    ?:  (gth her okay)
+      =.  saga.net  dex+her
+      go-core
+    =.  saga.net  lev+~
+    =.  cor  (watch-epic p.flag)
+    go-core
   ::
   ++  go-take-update
     |=  =sign:agent:gall
     ^+  go-core
     ?+    -.sign  (go-sub |)
-      %kick  (go-sub |)
+        %kick
+      ?>  ?=(%sub -.net)
```

Comment:
actually, you can't do this my bad.

The correct fix here is to remove the %load case in `$net` which will make this assertion correct, and just add a flag in the `%sub` case, to determine whether or not it has been loaded

Reply:
this will also need to be propagated to the other agents, inc. chat (mea culpa)

Reply:
done, with some lingering questions. no way to migrate from old %load subscriptions since we don't have a @p to use - unless we can grab it from somewhere else in the state.